Renaming Use, Migrate, MST and troubleshooting (#4193)
timescale/docs: use-timescale/security/saml.md

# SAML authentication

> $CLOUD_LONG offers SAML authentication as part of its [Enterprise][enterprise-tier] offering. SAML (Security Assertion Markup Language) is an open standard for exchanging authentication and authorization data between parties. With SAML enabled $CLOUD_LONG customers will be able to log into their $ACCOUNT_LONG using their existing SSO service provider credentials. 

Reviewer: ```suggestion
$CLOUD_LONG offers SAML authentication as part of its [Enterprise][enterprise-tier] offering. SAML (Security Assertion Markup Language) is an open standard for exchanging authentication and authorization data between parties. With SAML enabled $CLOUD_LONG customers can log into their $ACCOUNT_LONG using their existing SSO service provider credentials. 
```
